pkg/cluster/deploystorage.go

```diff
@@ -247,21 +247,32 @@ func (m *manager) deployStorageTemplate(ctx context.Context, installConfig *inst
 	return m.saveGraph(ctx, g)
 }
 
+var denyAssignmentsExclusions = map[string][]string{
+	"Microsoft.RedHatOpenShift/EnableSnapshots": {
+		"Microsoft.Compute/disks/beginGetAccess/action",
+		"Microsoft.Compute/disks/endGetAccess/action",
+		"Microsoft.Compute/disks/write",
+		"Microsoft.Compute/snapshots/beginGetAccess/action",
+		"Microsoft.Compute/snapshots/endGetAccess/action",
+		"Microsoft.Compute/snapshots/write",
+		"Microsoft.Compute/snapshots/delete",
+	},
+	"Microsoft.RedHatOpenShift/InspectEffectiveRouteTable": {
```

Comment:
There's a minor admin cost in setting up feature flags.  Let's name this one more vaguely so that it can be reused in the future for minor things like this - e.g. "Microsoft.RedHatOpenShift/RedHatEngineering"

Reply:
:+1:

Reply:
@jim-minter I don't think the suggested name makes much sense. IF this flag gonna be used by the customer to enable the ability to inspect EffectiveRouteTable, the name does not have any connection or meaning. Maybe something
`ElevatedDenyAssignment` or `ElevatedDebug` or smth under these lines more generic?

Reply:
I don't think the customer is - I think we're going to check it in one release cycle then release it to everyone afterwards.